storage/bucketmgr/main.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command bucketmgr is a program that manages Google Storage buckets by using
+// the Google Storage API. More documentation is available at
```

Comment:
It doesn't really manage anything because it's not interactive.

```
// Sample ... creates a bucket, lists buckets, and deletes a bucket using ...
```